src/main/steps/save-sign-out/content.ts

```diff
@@ -26,5 +28,10 @@ const languages = {
 };
 
 export const generateContent: TranslationFn = content => {
-  return languages[content.language](content);
+  const isInterimApplicationType =
```

Comment:
Nit: We could adjust the variable name slightly to show what it's checking
```suggestion
  const interimApplicationInProgress =
```